src/pages/[slug]/[ie]/index.tsx

```diff
@@ -188,13 +191,13 @@ const ObjectDetailPage: NextPage = () => {
 	}, [relatedData]);
 
 	/**
-	 * Variables
+	 * Computed
 	 */
 	const expandMetadata = activeTab === ObjectDetailTabs.Metadata;
 	const showFragmentSlider = mediaInfo?.representations && mediaInfo?.representations.length > 1;
 	const accessEndDate =
 		visitStatus && visitStatus.endAt ? formatMediumDateWithTime(asDate(visitStatus.endAt)) : '';
-
+	const isKioskUser = user?.groupName === 'KIOSK_VISITOR';
```

Comment:
DISCUSSION: Do we want to check this by user_group or make a permission: SEE_TOS_CALLOUT?

Reply:
Permissions for everything imo, keep it consistent even if it takes slightly more computation.

Also; abstract this to the user store as a selector.

Reply:
Helemaal akkoord. Fixed